src/qt/networkstyle.cpp

```diff
@@ -80,14 +80,12 @@ NetworkStyle::NetworkStyle(const QString &_appName, const int iconColorHueShift,
 const NetworkStyle* NetworkStyle::instantiate(const std::string& networkId)
 {
     std::string titleAddText = networkId == CBaseChainParams::MAIN ? "" : strprintf("[%s]", networkId);
-    for (unsigned x=0; x<std::size(network_styles); ++x)
```

Comment:
Would be nice to re-order the commits, so that the same line of code isn't changed twice in the same pull

Reply:
Good idea, done.